scripts/gen_mocks.sh

```diff
@@ -24,4 +24,4 @@ mockgen -destination=../mocks/amazon-vcp-resource-controller-k8s/pkg/pool/mock_p
 # package resource maocks
 mockgen -destination=../mocks/amazon-vcp-resource-controller-k8s/pkg/resource/mock_resources.go github.com/aws/amazon-vpc-resource-controller-k8s/pkg/resource ResourceManager
 # package condition maocks
-mockgen -destination=../mocks/amazon-vcp-resource-controller-k8s/pkg/condition/mock_condtion.go github.com/aws/amazon-vpc-resource-controller-k8s/pkg/condition Conditions
+mockgen -destination=../mocks/amazon-vcp-resource-controller-k8s/pkg/condition/mock_condition.go github.com/aws/amazon-vpc-resource-controller-k8s/pkg/condition Conditions
```

Comment:
Is this called from anywhere in automation or tests? I am not sure if you want to make this chanege without including the impacted files in this PR.

Reply:
`mock_condtion.go` was renamed to `mock_condition.go` in #166,  and `mock_condtion` has no references in the repo- I think this is safe to commit.